plugins/backend/rag-ai-backend/src/plugin.ts

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2024 Larder Software Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { loggerToWinstonLogger } from '@backstage/backend-common';
+import {
+  createBackendPlugin,
+  coreServices,
+  createExtensionPoint,
+} from '@backstage/backend-plugin-api';
+import { createRouter } from './service/router';
+
+import { AugmentationIndexer, RetrievalPipeline } from '@roadiehq/rag-ai-node';
+import { BaseLLM } from '@langchain/core/language_models/llms';
+
+export interface AugmentationIndexerExtensionPoint {
+  setAugmentationIndexer(augmentationIndexer: AugmentationIndexer): void;
+}
+
+export const augmentationIndexerExtensionPoint =
```

Comment:
The extension points should live in `-node` plugin so that modules using these extensions don't have to depend on the backend plugin

Reply:
Thanks, @drodil. Done. I was hoping that would fix the failing build, but it didn't seem to help.